Added a dashboard to user#show page

cacauzen opened the pull request:
We a meeting up tonight to talk about Gravtastic and usernames.

cacauzen commented:
![screenshot 2016-04-20 19 23 01](https://cloud.githubusercontent.com/assets/7612985/14693774/0861a68a-0730-11e6-8f71-9da95bd1509b.png)

kenmazaika commented:
Nice avatar.  Lol.

kenmazaika commented on app/assets/stylesheets/master.scss:
```diff
@@ -236,4 +235,18 @@ p {
+  border: 5px solid #AAAAAA;
+  height: 150px;
+  width: 150px;
+  background:url(trump.jpg) center;
```
I think we need to approach this differently if we want users to have custom uploaded avatars.  Similar to how we have photo uploading in nomster lesson 39.

cacauzen replied:
I know, just wanted to make sure it was responsive. So basically we are adding something like <%=  user.picture %>  right?

NKHolton replied:
I'm confused..I have the adding the user photo/avatar issue but it looks like you're already starting stuff on it, @cacauzen?  Please clarify because I'd love to get some assistance via pair programming on this but do the coding majority of on my own so I can continue to learn.  Let me know if I have it wrong though.

kenmazaika replied:
Ah, cool.  If this is a temporary placeholder, I can merge this and then we can make a separate ticket for implementing the avatar.  Does that make sense?

kenmazaika replied:
Ah, that makes sense.  This is related to the avatar uploading, but it's only fleshing out the feature, so you still have the ticket to work on, @NKHolton, but it might make sense to work with @cacauzen on image uploading since she was working on this code which is related.

NKHolton replied:
Yeah I already have an issue open for adding an avatar.  My thought was to use the gravtastic gem but I've reached out to the group and hopefully going to pair program after our group meeting tomorrow.

cacauzen replied:
I added this because I thought it would save us time to already have a  template to add code to, @NKHolton. None of the Gravatar stuff is done at all. We are doing that tomorrow! 👯

kenmazaika commented:
Overall it looks good.  I just made one comment.

amarkpark commented:
Is there a reason you're pushing all the .DS_Store files? Perhaps git ignore them?

cacauzen commented:
![screenshot 2016-04-21 11 51 53](https://cloud.githubusercontent.com/assets/7612985/14715678/9e9f102e-07b7-11e6-964a-1a01b88d57f8.png)

cacauzen commented:
Now trying to make the accordion collapse and expand.

kenmazaika commented:
Nice!  Let us know when you think it's ready to merge.

cacauzen commented:
@kenmazaika - @amarkpark suggested I added a helper class to one of the models, but someone mentioned in the video from last night's meeting that some tests are broken and I didn't want to interfere, so I just added code to the view and it's working perfect. @thank you @ampark for the suggestion, and also for fixing the layout and footer! It was driving me crazy!

kenmazaika commented:
@cacauzen is this ready to merge?  I think it is, but want to double check.

cacauzen commented:
Yes! I thought this moment would never come haha

cacauzen commented:
![screenshot 2016-04-22 11 16 14](https://cloud.githubusercontent.com/assets/7612985/14746033/b23363be-087b-11e6-9bbe-454c0233dbc9.png)

risu-kun commented:
Looks like the accordion function is working perfectly! Just some UX thoughts that can be made into separate issues--

1. What do you think of listing all the sections on page load, and having the actual post links be in accordions under the sections? I think that would result in less clicking to do for the user. Kind of like this:
![categories-index](https://cloud.githubusercontent.com/assets/3732709/14746496/eb3c5268-087d-11e6-9826-bc104d4aea95.png)

2. Related:  It looks like right now, the Section Title links are directing us to `post_path(section.id)`, which will be the show page for the post with the same id as the section. I don't think that's the behavior you wanted? ;) But this could be resolved by implementing something closer to #1.
 Thoughts?

kenmazaika commented:
I like the idea.  Let's make new tickets to address those ideas @risu-kun :+1:

cacauzen commented:
Yes, love it. It's basically nesting an accordion inside the existing accordion.

risu-kun commented:
We even just have the section titles stay there, in a non-accordion fashion, haha. If that's easier.

cacauzen commented:
Although, if you imagine a Category that has more than 10 sections, it might make it just as difficult for the user to find what they are looking for, since each section will have they own respective posts, it could be 10 posts... so I wonder which is worse, the clicking or the scrolling and reading...
